tests/snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py

```diff
@@ -264,6 +288,12 @@ def test_pfcwd_basic_single_lossless_prio_service_restart(snappi_api,
             logger.info("Wait until the system is stable")
             pytest_assert(wait_until(WAIT_TIME, INTERVAL, 0, duthost.critical_services_fully_started),
                           "Not all critical services are fully started")
+            pytest_assert(wait_until(WAIT_TIME, INTERVAL, 0, check_interface_status_of_up_ports, duthost),
+                          "Not all interfaces are up.")
+
+        # Wait for internal bgp to come up.
+        time.sleep(100)
```

Comment:
can you change this hard code value to use wait_util()?

here is an example:
https://github.com/sonic-net/sonic-mgmt/blob/72890a4a4e475a255b263a746030bc851426ae21/tests/bgp/test_bgp_session.py#L87

Reply:
Thanks for the examples. I am not sure if they work with some of the BGP sessions down. In our case, the backplane BGP are supposed to come up, but the front-facing(or ixia facing) BGP sessions are not brought up. I have to check if the above function can take only the backplane bgp neighbors.

Reply:
@rraghav-cisco you can use 'INTERNAL' to filter only iBGP sessions.

https://github.com/sonic-net/sonic-mgmt/blob/eb0081706b5166d4fa36255e7eb9504eedb32901/tests/bgp/test_bgp_session_flap.py#L70

Reply:
@sdszhang : Done.